### PR TITLE
Adding "most part of" relations to BSPO

### PR DESCRIPTION
**Pull request by megbalk**
issue #29

### CODE REVIEW

**Comment by matentzn** [comment]
I would like to get at least a rough review from @dosumis and @balhoff which pertains to the problems of property proliferation and basic modelling.

**Comment by megbalk** [comment]
@matentzn It doesn't allow me to merge, could you merge for me or give me permissions, please?

**Comment by matentzn** [comment]
@megbalk did we get any feedback for this from @cmungall ?

**Review by balhoff — CHANGES_REQUESTED** [review]
I generally think the relations sound good. @megbalk could you take a stab at providing textual definitions? Something like, e.g.,  for 'anteriormost part of': x 'anteriormost part of y' if x part of y and there is no z part of y that is more anterior than x and not part of x. Something like that.

**Comment by megbalk** [comment]
@balhoff I don't think I have the syntax correct, but what do you think?

File: src/ontology/bspo-edit.owl [hunk 1]
```diff
@@ -991,6 +996,69 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/BSPO_0020002> "ha
 ObjectPropertyDomain(<http://purl.obolibrary.org/obo/BSPO_0020002> <http://purl.obolibrary.org/obo/BSPO_0000010>)
 ObjectPropertyRange(<http://purl.obolibrary.org/obo/BSPO_0020002> <http://purl.obolibrary.org/obo/CARO_0000000>)
 
+# Object Property: <http://purl.obolibrary.org/obo/BSPO_0030003> (dorsalmost part of)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://orcid.org/0000-0003-2699-3066") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/BSPO_0030003> "x dorsalmost_part_of y and x part_of y 
+no z more dorsal than x
+x is adjacent to the dorsal boundary of y")
```
Comment (balhoff) [inline src/ontology/bspo-edit.owl:1003]: I haven't written too many of these relation definitions, but here is a stab at it.

```suggestion
AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://orcid.org/0000-0003-2699-3066") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/BSPO_0030003> "x dorsalmost_part_of y iff x part_of y and there is no z which is part_of y and is more dorsal than x but not part_of x; x is adjacent to the dorsal boundary of y")
```

Comment (balhoff) [reply src/ontology/bspo-edit.owl:1003]: If that sounds good to you, you could update the others to match.

**Review by balhoff — CHANGES_REQUESTED** [review]

File: src/ontology/bspo-edit.owl (hunk 1 above)
Comment (megbalk) [reply src/ontology/bspo-edit.owl]: What does "iff" mean?

Comment (balhoff) [reply src/ontology/bspo-edit.owl]: "If and only if"

**Comment by matentzn** [comment]
@balhoff did you review this PR again?

**Comment by matentzn** [comment]
I am asking so we can, or no, include it in the BSPO release we are about to produce.

**Review by balhoff — APPROVED** [review]

**Pull request merged**